program/CORE-V IDE prelminary project proposal.md

```diff
@@ -0,0 +1,251 @@
+
+For the above reasons, OpenHW is best placed to do this needed work.
+
+## Industry landscape: description of competing, alternative, or related efforts in the industry
+
+### Related efforts to be described
+
+- Eclipse CDT 
+- (GNU MCU) Eclipse Embedded CDT (Describe support for existing RISC-V devices)
+- Eclipse Theia
+
+## External dependencies
+ 
+The first build of CORE-V CDT builds on the following:
+- Eclipse Platform 2020-09
+- Eclipse CDT 10.0
+- Other components from Eclipse SimRel 2020-09
+- A suitable tool chain (see options below)
+
+The CORE-V IDE will be capable of integration with the following tool chain options. As per the project requirements above, the main focus of the CORE-V IDE will be the OpenHW CORE-V reference toolchain.
+
+### PULP Code Base As-Is
+
+This is a research tool chain from ETHZ and U of Bologna which has the following attributes:
+- It is updated often as research projects need.
+- A precompiled version of that toolchain is available, compiled and tested from Embecosm, updated weekly.
+- Drawbacks are that it is out of date (based on earlier release of GCC), doesn't meet coding standards, and lacks testing
+
+
+
+### Standard RISC-V tool chain
+This is the standard RISC-V GCC tool chain whose attributes include:
+- The development is hosted by the FSF (Free software foundation). It is completely up to date with GCC and binutils. There are many commits per day.
+- It has no Pulp features
+- A precompiled version of that toolchain is available from Embecosm and possibly others. From Embecosm, the latest version is updated weekly. The latest table release is 10.2.0
+- It is currently used in the Ashling IDE product.
+
+### OpenHW CORE-V Reference Tool Chain
+This is a separate project in OpenHW managed within the SW TG. 
+- The project starts from the standard GCC tools (FSF) and adds CORE-V instructions
+- The binutils-gdb repository in OpenHW github currently has 2 branches
+ - Stable (currently, this repository is a mirror of FSF).
+ - Development (first commits to this repository will have support for CORE-V hardware loop)
+
+
+
+
+## List of project outputs
+
+*Need a list of outputs, i.e.*
+- *Documents to be produced*
+- *Downloadable code to be produced*
+
+
+## TGs Impacted/Resource requirements
+
+This work will be planned and discussed in the SW TG. The project plan will be produced by the project manager with assistance of SW TG, and reviewed/approved in TWG
+
+
+## OpenHW engineering staff resource plan: requirement and availability
+
+- Florian Zaruba - technical contribution and project planning
+- Duncan Bees - program management oversight
+
+## Engineering resource supplied by members - requirement and availability
+
+*The intent here is to write down specific time/person requirements. This is not yet available, and will be supplied in the project plan. As of Preliminary Project Proposal, the major gap identified is a resource for an initial application to test the IDE.*

```

Comment:
I thought we had some explicit commitments from Alexander and Ashling?

Reply:
specific commitments moved from summary section to this section